buf/registry/module/v1beta1/module.proto

```diff
@@ -38,10 +38,10 @@ message Module {
   // The name of the Module.
   //
   // Unique within a given User or Organization.
-  string name = 4 [
-    (buf.validate.field).required = true,
-    (buf.validate.field).string.max_len = 255
-  ];
+  string name = 4 [(buf.validate.field).string = {
+    min_len: 2,
+    max_len: 100
```

Comment:
Are we sure we want to increase the user or organization name to 100? E.g., GH limits this to 39 chars iirc.

I think module makes sense at 100.